### ISSUE TITLE
Dragging orcaslicer window is extremely slow

### ISSUE THREAD

**Issue by Nuc1eoN** [SoftFever/OrcaSlicer#3111]
### OrcaSlicer Version

1.8.1

### OS version

Win 10 x64 22H2 build 19045.3803

### Additional system information

AMD RX580, Ryzen 7 3700x

### How to reproduce

EDIT: Set your mouse polling rate to 250hz. This bug only happens on high polling rate mice.

1. Open orcalslicer
2. Drag the window by the title bar

### Actual results

The window will drag VERY slowly.

### Expected results

Window dragging should not be laggy or unresponsive

If no-one can reproduce I can make a video.

**Comment by subutay** [comment]
I think so too. Perhaps it's due to the msedgewebview2 component, which is used to display printer web pages. However, this might not be a good idea

**Comment by RacerX10** [comment]
I had the same problem under Win 10, recently upgraded to Win 11 and the problem went away.  Might be helpful to troubleshoot.

**Comment by Nuc1eoN** [comment]
So looks like it only happens on Win10 so far.

**Comment by Sortam** [comment]
I see no such issue. Windows 10, Version 10.0.19044.2965

**Comment by jum0n** [comment]
Just installed it and have the same slow window move issue. Microsoft Windows [Version 10.0.19045.3803]

**Comment by Nuc1eoN** [comment]
@jum0n I'm also on Win10 22H2 build 19045.3803.

@Sortam interesting. Maybe it doesn't happen on older Win10 versions.

**Comment by Sortam** [comment]
@Nuc1eoN Yes, 64 bit (32-bit is a very rare beast these days, I would say :) ). 
I use a laptop with Intel Core i5-1035G4 CPU and its embedded GPU Iris Plus Graphics at screen-native 1920x1080 resolution. 
The laptop has 16 GB of RAM and run from SSD. It's nothing special, and it's obviously not the bleeding edge of technology...

**Comment by Nuc1eoN** [comment]
I just tested  this on my laptop which also has recent amd cpu/gpu hardware, but Orcaslicer behaves fine.
My laptop is on Win10 Home 19045.3636, so maybe I could try to update and see if it will break orcaslicer.

Another observation is that the bug only happens for the main Orcaslicer window. So if I open settings or another popup window it will behave fine.

Taskmanager performance tab does not show anything significant.

**Comment by Iarann** [comment]
Same behavior here. Win 10 x64, Intel i7-6700k, Nvidia GeForce RTX 2060

**Comment by Nuc1eoN** [comment]
> Same behavior here. Win 10 x64, Intel i7-6700k, Nvidia GeForce RTX 2060

Thank you for sharing, so it does not seem related to specific hardware as I have an AMD system.

Would you mind posting your Win10 build version? It should be under Settings -> System -> Info

**Comment by Iarann** [comment]
Windows build info:
Edition	Windows 10 Pro
Version	22H2
OS build	19045.3803
Experience	Windows Feature Experience Pack 1000.19053.1000.0

FYI, another piece of info is that Bambu Studio 1.8.2.56 has the exact same behavior on my system.

**Comment by Nuc1eoN** [comment]
@AlexanderS Wow thanks the bit about bambu studio being affected is new!

Also it seems that only Win10 22H2 19045.3803 is affected so far. Prior versions work fine.

Also found https://github.com/bambulab/BambuStudio/issues/934 and https://www.reddit.com/r/BambuLab/comments/15uocss/does_anyone_elses_bambu_studio_application_window/

**Comment by Nuc1eoN** [comment]
Hi so I have figured out what causes the issue.

It seems that it only happens when your mouse is set to a high polling rate like 1000hz.

Try to set the pollings rate down to the lowest setting and reproduce. I would be very thankful for feedback whether this helps for you too!

PS: Had the polling rate clue from this reddit post: https://www.reddit.com/r/WindowsHelp/comments/p9gwel/slow_motion_windowdragging_bug_on_windows_1011/

**Comment by Iarann** [comment]
> It seems that it only happens when your mouse is set to a high polling rate like 1000hz (gaming mice).

@Nuc1eoN Apparently for my system, "high polling rate" is anything about 125hz. Anything higher and I still have "inertia" issues where the screen keeps moving after the mouse stops. But, if I set it for 125hz it eliminates the "inertia", but I still see erratic behavior if I move the screen too quickly.

**Comment by CypisPiotr** [comment]
Same problem here (AMD Ryzen 5700G, Win10) - using Logitech G600 mouse,
polling at 250 and more Hz causes the drag on Orca window

**Comment by elyograg** [comment]
I'm having a similar problem in Windows 11... but it's with the middle button, dragging the view around. It drags SUPER slow. Rotating the view with the primary mouse button is fast.

**Comment by CypisPiotr** [comment]
Still present in 2.0.0

**Comment by dezenten** [comment]
Same problem here, maybe I need to upgrade to a faster computer? 
My setup: Windows 10 22H2, Intel I9-12900KS @5Ghz, Nvidia 3090.

**Comment by jum0n** [comment]
> Same problem here, maybe I need to upgrade to a faster computer? My setup: Windows 10 22H2, Intel I9-12900KS @5ghz, Nvidia 3090.

You probably need to upgrade to 4090.  The 3090 is known to be slow at melting filament.  :P

**Comment by Duglum** [comment]
Can confirm, it's the Mouse polling rate. 1000hz is super slow and sluggish, 125hz works as expected. Bambu Slicer has the same issue while Prusa Slicer works perfectly fine.

Win10 22H2 19045.4780
Orca Slicer v2.2.0-dev nightly from today.
AMD 7950X3D, Nvidia RTX 4090. All drivers are up to date.

This is extremely annoying, hope it will get fixed soon?

**Comment by MatthewMartinFD** [comment]
Any update on this? I'm seeing the same behavior.

**Comment by sgghostrider** [comment]
I came here from google, just installed OrcaSlicer for the first time and immediately searched it as soon as i saw it. It has been almost a year since the opening of this issue, any plans on fixing this? is this issue even known by the developers?

**Comment by DoubleStrike** [comment]
I have a feeling it's the interaction of not just a high mouse rate but also a refresh rate higher than 60 Hz.  I found a similar issue in Blender when dragging the window with the splash screen showing.  This really does need to be elevated in priority, as it is a really frustrating user experience

**Comment by jum0n** [comment]
After reducing my mouse polling rate, the issue went away for me.  I'm at 144mhz for display refresh, no issues.

**Comment by radry** [comment]
Issue still persists in 2.3.1 and Windows 10

Lowering Mouse polling rate just because one single program is broken is not a solution.

**Comment by otede** [comment]
Confirmed, stable 2.3.x, Windows 10 latest x64. What the hell is that supposed to be?! A program that flips because it can't handle high mouse polling rate?!

**Comment by NovelPhacelia** [comment]
Confirming this issue on Windows 10 21H2 x64, slicer version 2.3.1. Problem disappears when setting my mouse polling rate to 125

**Comment by DoubleStrike** [comment]
We definitely need to figure out how to solve this problem as the previously stated workarounds of lowering mouse polling rate or display refresh rate for one broken app is not even close to a valid solution. So let's ask, what logging or data capture will help the devs to narrow down the root cause here?

**Comment by radry** [comment]
@DoubleStrike Honestly I think the "root cause" is probably well known to the Devs. I have never experienced this behaviour in any program of the past 30 years however badly coded. This must be caused by a certain library they use for the GUI.

**Comment by Mic0de** [comment]
I still continue to have this issue. Changing polling rate does not do anything. I only have this issue with this one single application.
